### ISSUE TITLE
Exception: Upload session failed too many times(How can I remove scopes from my app?)

### ISSUE THREAD

**Issue by thiagofacanha** [OneDrive/onedrive-sdk-android#120]
Hi.
Im having a problem when uploading a file in Android using onedrive API.
The application was working normally but last week started to receive a failure with the exception informing that Upload session failed too many times for some accounts(Others are working without problem).
I did a test in one of these accounts removing the app permission in the OneDrive site and the application started to work again(after asking for permission), but cant do this programmatically.
Someone knows what could have happened?

Im receiveing in the failure result of upload callback the com.onedrive.sdk.core.ClientException with that message "Upload session failed to many times."
Inside ChukedUploadRequest.java the error that its receiveing is Bad Request.

**Comment by thiagofacanha** [comment]
I notted something. 
The accounts that arent working with the app have more permissions  than the ones that are working.
In the beginning of the app i used the previous version of OneDrive API( skydrive)
I tested putting the older scopes with the new ones
("wl.signin","wl.basic","wl.skydrive","wl.offline_access","wl.skydrive_update")
and now the same problem happens.
So probably my problem is with these old scopes...
Is there any way of remove programmatically a scope from an app in OneDrive?
Thxs
